### ISSUE TITLE
Uncaught DataCloneError: Failed to execute 'postMessage' on 'Worker': An object could not be cloned.

### ISSUE THREAD

**Issue by Cortys** [tststs/atom-ternjs#202]
Occurs as soon as the tern server is queried the first time (caused by the tooltip as well as the autocomplete feature).

**Atom Version**: 1.4.0
**System**: Mac OS X 10.11.2
**Thrown From**: [atom-ternjs](https://github.com/tststs/atom-ternjs) package, v0.13.1

### Stack Trace

Uncaught DataCloneError: Failed to execute 'postMessage' on 'Worker': An object could not be cloned.

```
At /Users/admin/.atom/packages/atom-ternjs/lib/atom-ternjs-server.js:146

Error: Failed to execute 'postMessage' on 'Worker': An object could not be cloned.
    at Error (native)
    at ReadFileContext.callback (/Users/admin/.atom/packages/atom-ternjs/lib/atom-ternjs-server.js:146:23)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:303:13)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "turbo-javascript",
      "autoflow",
      "wrap-guide",
      "autoclose-html",
      "sublime-tabs",
      "highlight-line",
      "scope-inspector",
      "welcome",
      "markdown-scroll-sync",
      "language-javascript-semantic",
      "node-debugger",
      "symbols-tree-view",
      "emmet",
      "tree-view-open-files"
    ],
    "projectHome": "/Users/admin/Documents",
    "themes": [
      "quantum-ui",
      "one-dark-syntax"
    ]
  },
  "atom-ternjs": {
    "caseInsensitive": false,
    "guess": false,
    "useSnippets": true,
    "useSnippetsAndFunction": true
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.2.1
Sublime-Style-Column-Selection, v1.5.1
atom-ternjs, v0.13.1
autocomplete-modules, v1.3.1
autocomplete-paths, v1.0.2
autocomplete-php, v0.3.7
docblockr, v0.7.3
editorconfig, v1.2.4
file-icons, v1.6.14
hex, v0.6.2
highlight-selected, v0.11.1
keyboard-localization, v1.4.14
language-groovy, v0.5.0
language-latex, v0.6.1
language-mips, v0.2.0
language-prolog, v0.10.0
language-sml, v0.5.0
latex, v0.28.2
latexer, v0.2.7
less-than-slash, v0.12.0
linter, v1.11.3
linter-chktex, v1.2.4
linter-clang, v3.3.0
linter-eslint, v5.2.7
linter-javac, v1.5.0
linter-jscs, v3.4.0
linter-jshint, v1.2.2
linter-jsonlint, v1.1.4
minimap, v4.19.0
minimap-find-and-replace, v4.4.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-linter, v1.1.1
pdf-view, v0.39.0
pigments, v0.19.7
project-manager, v2.7.3
quantum-ui, v0.2.2
remote-sync, v3.5.0

# Dev
No dev packages
```

**Comment by tststs** [comment]
Had the same issue 5 minutes ago.
Happens if a file is not found and the server tries to send the err object from `fs.readfile` via `sendMessage` which can't be cloned. Should be fixed with 0.13.2.

Greetings

**Comment by Cortys** [comment]
Awesome! Works great now.

Thanks for that blazing fast fix.
